src/services/api.tsx

```diff
@@ -56,6 +56,8 @@ const useGetUrlsQuery = ({ enabled = true }: { enabled?: boolean }) => {
     return useQuery({
         queryKey: ['urls'],
         enabled: enabled,
+        retry: false,
```

Comment:
When does it retry?

Reply:
@vinit717

Reply:
Sorry I just remove this 
As retry is basically to again and again retry the query till the no we set before showing any error